stdlib/json.gr

```diff
@@ -0,0 +1,2103 @@
+/**
+ * JSON (JavaScript Object Notation) parsing, printing, and access utilities.
+ */
```

Comment:
Do you want to add the examples to the module header as you've been doing on the rest of stdlib?